src/utils/FileManagementUtils.js

```diff
@@ -2,14 +2,14 @@
 // Licensed under the MIT license.
 
 import { TABLE_DATA_STATUS, UPLOAD_FILE_STATUS_KEY } from '@cosmotech/ui';
-import DatasetService from '../../services/dataset/DatasetService';
-import WorkspaceService from '../../services/workspace/WorkspaceService';
-import { AppInsights } from '../../services/AppInsights';
-import { DATASET_ID_VARTYPE } from '../../services/config/ApiConstants';
-import { ConfigUtils, DatasetsUtils, ScenarioParametersUtils } from '../../utils';
-import applicationStore from '../../state/Store.config';
+import DatasetService from '../services/dataset/DatasetService';
+import WorkspaceService from '../services/workspace/WorkspaceService';
+import { AppInsights } from '../services/AppInsights';
+import { DATASET_ID_VARTYPE } from '../services/config/ApiConstants';
+import { ConfigUtils, DatasetsUtils, ScenarioParametersUtils } from '.';
```

Comment:
It seems we have a dependencies cycle between _FileManagementUtils.js_ and _ScenarioParametersUtils.js_, they are both importing each other.

Reply:
done